src/code/client.js

```diff
@@ -701,7 +713,16 @@ class CloudFileManagerClient {
   }
 
   isShared() {
-    return (this.state.currentContent != null ? this.state.currentContent.get("sharedDocumentId") : undefined) && !(this.state.currentContent != null ? this.state.currentContent.get("isUnshared") : undefined)
+    const currentContent = this.state?.currentContent
+    if(currentContent) {
+      const unshared = currentContent.get("usUnshared")
```

Comment:
I think that is supposed to be `isUnshared` instead of `usUnshared`

Reply:
Yikes -- thanks.